lib/features/mailbox_dashboard/domain/repository/email_cache_repository.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:jmap_dart_client/jmap/mail/email/email.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/data/model/email_cache.dart';
+
+abstract class EmailCacheRepository {
```

Comment:
We also have `EmailRepository`, how about it? Why we need this repository? if yes, how to optimize it?